WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -157,7 +160,14 @@ extension ImageAndTitleAndTextTableViewCell {
         descriptionLabel.numberOfLines = viewModel.numberOfLinesForText
         contentImageView.image = viewModel.image
         contentImageStackView.isHidden = viewModel.image == nil
-        accessoryType = viewModel.isActionable ? .disclosureIndicator : (viewModel.isSelected ? .checkmark : .none)
+        accessoryType = viewModel.showsDisclosureIndicator ? .disclosureIndicator : .none
+        if viewModel.isActionable && viewModel.showsDisclosureIndicator == false {
+            accessoryType = .none
+        } else if viewModel.isActionable == false && viewModel.showsDisclosureIndicator && viewModel.isSelected {
+            accessoryType = .checkmark
+        } else if viewModel.isActionable == false && viewModel.showsDisclosureIndicator && viewModel.isSelected == false {
+            accessoryType = .none
+        }
```

Comment:
Is there a cleaner way to write this bit? I found myself being very specific with the cases because we're using all of them across the app in a way that (seems to be) inconsistent manner, for example, we hide the disclosure indicator in Order Creation and show it on "Add Files" when both are actionable and start a new flow, so showing and indicator would be expected (or hide it in both).